### DISCUSSION TITLE
Generic FOSS documents update

### DISCUSSION THREAD

**Discussion by ekaterina-leonteva** [d3/d3#3639] [Q&A]
Hello!

I'm working at Ericsson and we are using d3.js in one of our products. Currently we are doing FOSS evaluation for the product and I received a feedback about `d3-d3-selection` and `d3-d3-zoom`: "Generic FOSS documents expired. That document is valid 18 months, a mail must be sent to 3PPT asking them to update them." This is bit of a showstopper for us as we can't release without this evaluation.

Please let me know if the update can be done.

**Comment by curran** [comment]
What change are you proposing? What file should be updated?

**Comment by mbostock** [comment]
I expect that the “generic FOSS documents” refer to Ericsson’s own documents, and certainly not anything of ours. I suggest you ask for help within Ericsson rather than here. We provide the free software, but it’s entirely your responsibility to decide whether you want to use it!

**Reply by ekaterina-leonteva** [reply]
Ok, thanks for the info. This bureaucracy is a bit of a nightmare, but I'll try to figure it out because we love d3 and want to continue using it 🤟
